runner/utils/auth/helpers.go

```diff
@@ -0,0 +1,12 @@
+package auth
+
+import "github.com/dgrijalva/jwt-go"
+
+func (m *Module) GenerateHS256Token(serviceId, projectId, version string) (string, error) {
```

Comment:
Rename to `GenerateTokenForRuntimeCode`. Also but this in a file named `operations.go`